### ISSUE TITLE
Add uptime indicator

### ISSUE THREAD

**Issue by gemal** [processhacker/processhacker#1053]
The original task manager has a Uptime indicator:
![image](https://user-images.githubusercontent.com/69105/145551763-3d9af39c-26a9-41e6-871b-433c816d1c09.png)

Process Hacker doesn't seem to have this

**Comment by dmex** [comment]
The uptime is on the system information window.

**Comment by gemal** [comment]
> The uptime is on the system information window.

I just cant see where...
Can you paste a screenshot

I'm using Process Hacker Version: 3.0.4399 (1ecc78c)

**Comment by dmex** [comment]
![image](https://user-images.githubusercontent.com/1306177/145861710-0f6094b9-21e0-48e3-afca-9962ce82d7d5.png)

**Comment by shodanx2** [comment]
I searched "everywhere", couldn't find it, logged in to github to note the strange omission and now I see it. 

Now it's obvious, I don't know why I couldn't find it !

<img width="731" height="851" alt="Image" src="https://github.com/user-attachments/assets/8659a804-e40e-4b81-b586-f95d8112b6de" />


Although one of my friend that I installed system informer told me, he couldn't find the performance section at all, until I pointed to him the "System Information" button. That too isn't a bit unexpected and I think would be better as a "tab".  Even if it's not a real tab, but just a button that opens the system information window

In any case this is solved for me !
